addons/sale_timesheet/views/sale_timesheet_portal_templates.xml

```diff
@@ -116,8 +116,7 @@
                         <t t-if="invoice_line.id in invoices_accessible">
                             <a t-attf-href="/my/invoices/{{ invoice_line.id }}">
                                 <i t-if="invoice_line.state == 'draft'">Draft Invoice</i>
-                                <t t-else="" t-esc="invoice_line.name"/>
-                            </a>
+                                <t t-else="" t-esc="invoice_line.name"/></a>
```

Comment:
Is there any reason for this changes? It seems useless, isn't it? 🤔

Reply:
@xavierbol , It's not useless, due to the new line extra space after the invoice name was added. so to remove it we did this change. Thanks

Reply:
Oh okay :o 
Thanks for the explanation. 🙂